content/docs/06-integrations/00-trident.md

```diff
@@ -0,0 +1,203 @@
+| ``tridentAutosupportImage`` | Allows the complete override of the image for Trident's Autosupport container. |
+| ``tridentAutosupportImageTag`` | Allows overriding the tag of the image for Trident's Autosupport container. Default: `23.01` |
+| ``tridentAutosupportProxy`` | Allows Trident's autosupport container to phone home via an HTTP proxy. |
+| ``tridentLogFormat`` | Sets the Trident logging format (text or json). Default: `text` |
+| ``tridentDisableAuditLog`` | Disables Trident's audit logger. Default: `true` |
+| ``tridentImage`` | Allows the complete override of the image for Trident. |
+| ``tridentImageTag`` | Allows overriding the tag of the image for Trident. |
+| ``tridentEnableNodePrep`` | (Deprecated) Attempts to automatically install required packages on nodes. Default: `false` |
+| ``tridentSkipK8sVersionCheck`` | (Deprecated) Allows overriding the k8s version limit for Trident. Default: `false` |
+| ``tridentProbePort`` | Allows overriding the default port used for k8s liveness/readiness probes. |
+| ``windows`` | Allows Trident to be installed on Windows worker node. Default: `false` |
+| ``enableForceDetach`` | Allows enabling the force detach feature. Default: `false` |
+| ``excludePodSecurityPolicy`` | Excludes the operator pod security policy from creation. Default: `false` |
+
+
+## Usage
+
+After deploying Trident, you will need to create a backend and a storage class before you can start provisioning volumes and mounting those to any pods. 
+
+<br />
+
+#### Create a backend.  
+
+Trident supports multiple backends and depending your storage backend, you will need to choose a backend that fits your needs. You can find example driver manifests [here](https://github.com/NetApp/trident/tree/master/trident-installer/sample-input/backends-samples) and you will need to personalise them with your credentials and environment configurations. Here's an example with the ONTAP-NAS driver. 
+
+<br />
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: backend-ontap-nas-secret
+  namespace: trident
+type: Opaque
+stringData:
+  username: [USERNAME]
+  password: [PASSWORD]
+---
+apiVersion: trident.netapp.io/v1
+kind: TridentBackendConfig
+metadata:
+  name: backend-ontap-nas
+  namespace: trident
+spec:
+  version: 1
+  storageDriverName: ontap-nas
+  managementLIF: [ONTAP_MANAGEMENT_LIF_IP]
+  dataLIF: [DATA_LIF_IP]
+  backendName: ontap-nas
+  autoExportCIDRs:
+  - [x.x.x.x/xx]
+  autoExportPolicy: true
+  svm: [SVM]
+  credentials: 
+    name: backend-ontap-nas-secret
+```
+
+<br />
+
+#### Create a storage class.  
+
+Kubernetes allows us to bind statically and/or dynamically provisioned volumes to our Pods. Statically provisioned volumes are manually created by a user and then referenced in a deployment. Astra Trident allows you to leverage your NetApp storage and 
+Before you can request dynamically provisioned volumes, you will need to create a storage class. Storage classes 
```

Comment:
```suggestion
Kubernetes supports the ability to bind statically or dynamically provisioned volumes to Pods. Statically provisioned volumes are manually created by a user and then referenced in a deployment. Astra Trident allows you to leverage your NetApp storage. You must create a storage class before you can request dynamically provisioned volumes. Storage classes 
```

@Sheldonwl This paragraph seems to be missing a sentence related to the "Astra Trident allows you to leverage your NetApp storage" sentence.